recipe/meta.yaml

```diff
@@ -274,7 +274,9 @@ outputs:
       files:
         - test-r-xgboost.r
       commands:
-        - Rscript test-r-xgboost.r  # [not win]
+        - $R -e "library('xgboost')"           # [unix]
+        - "\"%R%\" -e \"library('xgboost')\""  # [win]
+        - Rscript test-r-xgboost.r             # [unix]
```

Comment:
Copying over the `library(...)` tests so we have a smoke test

Provides some coverage on Windows while working out issue: https://github.com/conda-forge/xgboost-feedstock/issues/205